docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,115 @@
+- From the wrong account, go to **[Receipts](https://expo.dev/accounts/[account]/settings/receipts)** and initiate a [request for a refund](/billing/invoices-and-receipts/#request-a-refund).
+
+### I am on a Free plan and only need a few extra builds or updates
+
+If you are on a Free plan and have completed your monthly quota of free builds and updates, upgrade to the [On-demand plan](/billing/plans/). It offers a pay-as-you-go option for EAS Build and the same limits as the Free plan for EAS Update. Once your requirements are fulfilled, you can [downgrade to the Free plan from the On-demand plan](/billing/manage/#cancel-a-plan).
+
+To upgrade from the Free plan to the On-demand plan, see [Upgrade to a new plan](/billing/manage/#upgrade-to-a-new-plan).
+
+To downgrade from an On-demand to a Free plan, see [Cancel a plan](/billing/manage/#cancel-a-plan).
+
+## Billing
+
+### When does a billing period start for a plan?
+
+For the Free plan, the billing period starts on the first day of the calendar month.
+
+For [all paid plans](/billing/plans/#plans), the billing period starts from the subscription date of that plan.
+
+### How do I update my billing information or add a tax ID?
+
+To update your Organization account's billing information or add a tax ID, make sure that you have either an [Owner or Admin role](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role. After confirming your role:
+
+- Go to your [account's Billing](https://expo.dev/settings/billing), and click on **Update billing information**. This will take you to Stripe's portal.
+- On Stripe's portal, under **Billing information**, click on **Update information** to update billing-related information such as billing name, email, address, and tax ID.
+
+See [Manage billing information](/billing/manage/#manage-billing-information) for more details.
+
+### Can I update the billing information on my last invoice?
+
+No. Updating billing information will only be reflected on the next invoice.
+
+### Can you email me the receipts?
+
+No. Your account's Owner or Admin can download them. See [Download an invoice](/billing/invoices-and-receipts/#download-and-view--an-invoice) for more information.
+
+### How can I reduce the amount of EAS Build usage by using EAS Update?
+
+Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. Soon, using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime), you will be able to build or update selectively, depending on whether you already have a compatible native runtime.
```

Comment:
```suggestion
Use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without creating a new build. This option is better for most apps since JavaScript code changes more frequently than the underlying native code. You can create multiple test channels with EAS Update and reduce the need to create additional builds for your team. Soon, by using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime), you will be able to build or update selectively, depending on whether you already have a compatible native runtime.
```